src/WidgetBase.ts

```diff
@@ -198,13 +191,14 @@ export class WidgetBase<P = WidgetProperties, C extends DNode = DNode> extends E
 		this._properties = <P> {};
```

Comment:
DOH! Sorry, I think I remember that now.  😢 Is it worth it to PR maquette to use `Object.hasOwnProperty.call(props, key)` so that `Object.create(null)` can be used, or is it just a minimal improvement thing that it wouldn't matter?